sonic_data_client/db_client.go

```diff
@@ -79,9 +79,20 @@ var IntervalTicker = func(interval time.Duration) <-chan time.Time {
 	return time.After(interval)
 }
 
+var ExportTableData2Msi = tableData2Msi
```

Comment:
>tableData2Msi [](http://example.com/codeflow?start=26&length=13)

Is it better just to change to `TableData2Msi` ?

Reply:
I don't see an issue with changing it to a public function for multiple packages to use. Will change to public.